utils/helpers/alpaca-ui.js

```diff
@@ -0,0 +1,49 @@
+var alpacaUIMixin = {
+  computed: {
+    alpacaConfig () {
+      const config = this.$options.config
+      return config
```

Comment:
can you add an empty line before return?

Reply:
sure